lib/movable_ink/aws/route53.rb

```diff
@@ -10,6 +10,23 @@ def resource_record_sets(hosted_zone_id)
         @resource_record_sets[hosted_zone_id] ||= list_all_r53_resource_record_sets(hosted_zone_id)
       end
 
+      def get_resource_record_sets_by_instance_name(zone, instance_name)
+        resource_record_sets(zone).select{|rrs| rrs.set_identifier == instance_name}.first.to_h
```

Comment:
This feels somewhat brittle as it's assuming the `set_identifier` is equal to the `instance_name`, which is not an enforceable requirement.  I recognize the use case, but I'm wondering if there's a more general solution possible.